src/providers/wfs/qgswfsdescribefeaturetype.cpp

```diff
@@ -40,6 +40,8 @@ bool QgsWFSDescribeFeatureType::requestFeatureType( const QString &WFSVersion,
     }
   }
 
+  // Always add singular form for broken servers (ESRI)
```

Comment:
I don't think the comment is exact. ESRI supports plural form. This is other servers that don't support the plural form

Reply:
```suggestion
  // Always add singular form for broken servers
```